Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/DataContentViewerOtherCases.java

```diff
@@ -754,57 +734,139 @@ private void populateTable(Node node) {
+                    }
+                } else {
+                    try {
+                        dataSources.add(makeDataSourceString(Case.getCurrentCaseThrows().getName(), nodeData.getDeviceID(), nodeData.getDataSourceName()));
+                        caseNames.put(Case.getCurrentCaseThrows().getName(), new CorrelationCase(Case.getCurrentCaseThrows().getName(), Case.getCurrentCaseThrows().getDisplayName()));
+                    } catch (NoCurrentCaseException ex) {
+                        LOGGER.log(Level.WARNING, "No current case open for other occurrences");
+                    }
+                }
+                totalCount++;
+            }
         }
-
+        for (CorrelationCase corCase : caseNames.values()) {
+            casesTableModel.addCorrelationCase(new CorrelationCaseWrapper(corCase));
+        }
+        int caseCount = casesTableModel.getRowCount();
         if (correlationAttributes.isEmpty()) {
-            tableModel.addNodeData(new OtherOccurrenceNodeMessageData(Bundle.DataContentViewerOtherCases_table_noArtifacts()));
-            setColumnWidthToText(0, Bundle.DataContentViewerOtherCases_table_noArtifacts());
-        } else if (0 == tableModel.getRowCount()) {
-            tableModel.addNodeData(new OtherOccurrenceNodeMessageData(Bundle.DataContentViewerOtherCases_table_noResultsFound()));
-            setColumnWidthToText(0, Bundle.DataContentViewerOtherCases_table_noResultsFound());
-        } else {
-            setColumnWidths();
+            casesTableModel.addCorrelationCase(NO_ARTIFACTS_CASE);
+        } else if (caseCount == 0) {
+            casesTableModel.addCorrelationCase(NO_RESULTS_CASE);
         }
-        
+        setColumnWidths();
         setEarliestCaseDate();
-        setOccurrenceCounts();
+        foundInLabel.setText(String.format(Bundle.DataContentViewerOtherCases_foundIn_text(), totalCount, caseCount, dataSources.size()));
+        if (caseCount > 0) {
+            casesTable.setRowSelectionInterval(0, 0);
+        }
     }
 
-    /**
-     * Adjust a given column for the text provided.
-     *
-     * @param columnIndex The index of the column to adjust.
-     * @param text        The text whose length will be used to adjust the
-     *                    column width.
-     */
-    private void setColumnWidthToText(int columnIndex, String text) {
-        TableColumn column = otherCasesTable.getColumnModel().getColumn(columnIndex);
-        FontMetrics fontMetrics = otherCasesTable.getFontMetrics(otherCasesTable.getFont());
-        int stringWidth = fontMetrics.stringWidth(text);
-        column.setMinWidth(stringWidth + CELL_TEXT_WIDTH_PADDING);
+    private String makeDataSourceString(String caseUUID, String deviceId, String dataSourceName) {
+        return caseUUID + deviceId + dataSourceName;
+    }
+
+    private void updateCaseSelection() {
```

Comment:
Missing header doc

Reply:
Also could you consider a rename of the function? What it seems like its doing is updating the data sources list on a new case selection, which isn't immediately clear from the name updateCaseSelection()